src/css/Contact.css

```diff
@@ -0,0 +1,27 @@
+.reach{
+padding-top: 1%;
+padding-left: 27%;
+width: 35%;
+height: 23%;
+text-align: left;
+font-family: Arial;
+font-size:  10vh;
+line-height: 11vh;
+letter-spacing: 0;
+color: #FF4081;
+opacity: 1;
+}
+
+.number {
+padding-top: 10%;
+padding-left: 50%;
+width: 80%;
+height: 24%;
+text-align: left;
+font-family:Arial;
+font-size: 6vh;
```

Comment:
Change this font size to 6vh so that it resembles the references font size. Also we've changed the font to Lato.